internal/common/enums.rs

```diff
@@ -180,9 +180,10 @@ macro_rules! for_each_enums {
             /// The role for the `accessible-role` property
             enum AccessibleRole {
                 none,
-                text,
                 button,
                 checkbox,
+                tab,
+                text,
```

Comment:
This is documented in builtin_elements.md

(yeah, ideally we would have the documentation inline and we'd extract the enum documentation)